README.md

```diff
@@ -927,6 +927,8 @@ generateQueryEmbeddings().then((embeddingsResponse) => {
 
 ## Productionizing
 
+**Warning:** The Pinecone SDK is intended for server-side use only. Using the SDK within a browser context in a production setting could lead to exposing your API key to third parties. If you have deployed the SDK to production in a browser, please rotate your API keys.
```

Comment:
Total and complete nit, but I'd phrase this as a `Note` rather than a warning. I feel like `warning` implies there's something bad about this behavior, when in fact it's preferred behavior. Might just be me though!

Reply:
Just to make this a bit more direct, instead of 

> Using the SDK within a browser context in a production setting could lead to exposing your API key to third parties.

I'd say

> Using the SDK within a browser context can expose your API key(s).